src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -98,7 +109,7 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
       className={classnames('breadcrumb--container', {
         'breadcrumb--container__overflowfull': afterOverflowItems.length === 1,
       })}
-      ref={breadcrumbRef}
+      ref={typeof ResizeObserver !== 'undefined' ? breadcrumbRef : null}
```

Comment:
nit: We check this twice now, perhaps put in a function and call it like `browserSupports(ResizeObserver)` to make it  clear that we are dealing with feature detection.

Reply:
👍 Great suggestion, done. 

I took a trip down memory lane to see if we could/should use Modernizr but they don't support `esm` and also don't have a feature detection check for `ResizeObserver`. The function is extensible for future detection.

Reply:
Sweet!